crates/tauri-utils/src/acl/schema.rs

```diff
@@ -72,14 +88,28 @@ pub trait PermissionSchemaGenerator<
     let mut permission_schemas = Vec::new();
 
     // schema for default set
-    if self.has_default_permission_set() {
-      let default = Self::perm_id_schema(name, "default", self.default_set_description());
+    if let (Some(description), Some(permissions)) = (
+      self.default_set_description(),
+      self.default_set_permissions(),
+    ) {
+      let default = Self::perm_id_schema(
+        name,
+        "default",
+        Some(&add_permissions_to_description(description, permissions)),
+      );
```

Comment:
for the sake of readability, let's break this a bit
```suggestion
      let description = add_permissions_to_description(description, permissions);
      let default = Self::perm_id_schema(name, "default", Some(&description));
```